hardhat.config.ts

```diff
@@ -0,0 +1,35 @@
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@nomiclabs/hardhat-ethers";
+
+// The next line is part of the sample project, you don't need it in your
+// project. It imports a Hardhat task definition, that can be used for
+// testing the frontend.
+import "./tasks/accounts.ts";
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.17",
+  networks: {
+    hardhat: {
+      chainId: 31337, // We set 1337 to make interacting with MetaMask simpler
+      accounts: {
+        mnemonic: "test test test test test test test test test test test junk",
+        path: "m/44'/60'/0'/0",
+        initialIndex: 0,
+        count: 20,
+        passphrase: "",
+      },
+    },
+    GSChain: {
```

Comment:
Remove GSChain to DSChain
chainId to 326
mnemonic to something else

Reply:
Should I remove it or keep it? In the previous comment you suggested that we should remove from line 18-29

Reply:
change it to "dog dog dog ... dog dog"